exercises/03-sagas/README.md

```diff
@@ -32,6 +32,8 @@ For more info, please see [the instructions for running the exercise solutions](
 
 At the moment we gather all successfully paid orders and the only option we have is to process them in batches. Business is growing rapidly and we would like to process orders as soon as possible, not just at predefined intervals.
 
+Note: This is a hypothetical case. From experience we've seen a lot of developers implement a batch processes. Sagas are a better alternative to batch processes. For more info, read our blog post, "[Death to the batchjob](https://particular.net/blog/death-to-the-batch-job)".
```

Comment:
- "...implement a batch processes" - remove "a"
- "batchjob" - should be "batch job"